vertx-pg-client/src/main/java/io/vertx/pgclient/impl/codec/DataType.java

```diff
@@ -229,5 +220,8 @@ static DataType lookup(Class<?> type) {
     encodingTypeToDataType.put(Polygon[].class, POLYGON_ARRAY);
     encodingTypeToDataType.put(Circle.class, CIRCLE);
     encodingTypeToDataType.put(Circle[].class, CIRCLE_ARRAY);
+
+//    encodingTypeToDataType.put(String.class, XML);
```

Comment:
why is this commented ?

Reply:
See the Comment from the 22. of Dec:

` So i did it using String now and it works just fine i think - one problem i encountered was the mapping in the DataType.java because there is already a mapping for String to Varchar

encodingTypeToDataType.put(String.class, VARCHAR); encodingTypeToDataType.put(String[].class, VARCHAR_ARRAY); encodingTypeToDataType.put(String.class, XML); encodingTypeToDataType.put(String[].class, XML_ARRAY);

Which seems to be only important in case of a PrepareStatementCommand with parameterTypes which the test i'm executing are apparently not using. Maybe you have some more insight into that?`